sys/shell/commands/sc_gnrc_6ctx.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <inttypes.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include "net/ipv6/addr.h"
+#include "net/gnrc/sixlowpan/ctx.h"
+#include "net/gnrc/sixlowpan/nd.h"
+#include "timex.h"
+#include "xtimer.h"
+
+static xtimer_t del_timer[GNRC_SIXLOWPAN_CTX_SIZE];
+static gnrc_sixlowpan_nd_router_abr_t *abr = NULL;
+
+void _del_cb(void *ptr)
+{
+    gnrc_sixlowpan_ctx_t *ctx = ptr;
+    uint8_t cid = ctx->flags_id & GNRC_SIXLOWPAN_CTX_FLAGS_CID_MASK;
+    ctx->prefix_len = 0;
+    gnrc_sixlowpan_nd_router_abr_rem_ctx(abr, cid);
+    del_timer[cid].callback = NULL;
+}
+
+static void _usage(char *cmd_str)
+{
+    printf("usage: %s [{add <0-15> <prefix>/<prefix_len> <ltime in min>|del <ctx>}]\n", cmd_str);
+    puts("       Note that `del` only will invalidate the context for compression.");
+    puts("       This way the network has time to get the context update dissiminated.");
+    puts("       It will be reassignable after 5 min.");
```

Comment:
I think you can shorten this to something like: "'del' will only invalidate the context for compression which can be reassigned after 5 min."